api/timeseries_storage_api.go

```diff
@@ -31,7 +31,6 @@ type FetchTimeseriesRequest struct {
 	Metric                TaggedMetric // metric to fetch.
 	SampleMethod          SampleMethod // up/downsampling behavior.
 	Timerange             Timerange    // time range to fetch data from.
-	MetricMetadata        MetricMetadataAPI
```

Comment:
The API data wasn't actually being used, so it has been removed. This had very little effect on existing code.